tribe-ext-list-venues-organizers-shortcodes.php

```diff
@@ -97,116 +103,162 @@ protected function format() {
 
 			while ( $this->query->have_posts() ) {
 				$this->query->the_post();
-				$shortcode_type = 'tribe_organizer' == $this->atts['post_type'] ? 'organizer' : 'venue';
-				$id             = get_the_ID();
+				$post_id = get_the_ID();
 
 				// count upcoming events
 				$args = array(
 					'start_date' => date( 'Y-m-d H:i:s' ),
 				);
 
-				if ( 'organizer' == $shortcode_type ) {
-					$args['organizer'] = $id;
-				} else {
-					$args['venue'] = $id;
+				if ( Tribe__Events__Main::ORGANIZER_POST_TYPE == $this->atts['post_type'] ) {
+					$args['organizer'] = $post_id;
+				} elseif ( Tribe__Events__Main::VENUE_POST_TYPE == $this->atts['post_type'] ) {
+					$args['venue'] = $post_id;
 				}
 
+				$args = apply_filters( 'tribe_ext_list_venues_organizers_shortcodes_tribe_get_events_per_post_args', $args, $post_id, $this->atts );
```

Comment:
Unless this was an existing hook that was relocated, can we shorten it a little by removing the second use of 'tribe_'?